app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,18 @@
+class AdminController < ActionController::Base
+  layout 'admin'
+  before_action :ensure_user_signed_in
+
+  helper_method :current_user
+
+  private
+
+  def current_user
+    session[:admin_account]
+  end
+
+  def ensure_user_signed_in
+    return if current_user.present? && ENV['ADMIN_EMAILS'].split(',').include?(current_user['email'])
```

Comment:
:+1:

This has the benefit that we can also revoke access if required.

Reply:
yup :-) Was a good idea.